app/models/file-version.ts

```diff
@@ -1,11 +1,20 @@
 import { attr } from '@ember-data/model';
+import { Link } from 'jsonapi-typescript';
+import { NormalLinks } from 'osf-api';
 
 import OsfModel from './osf-model';
 
+export interface FileVersionLinks extends NormalLinks {
+    download?: Link;
+    render?: Link;
+}
+
 export default class FileVersionModel extends OsfModel {
     @attr('number') size!: number;
     @attr('date') dateCreated!: Date;
     @attr('fixstring') contentType!: string;
+    @attr('fixstring') name!: string;
+    @attr() links!: FileVersionLinks;
```

Comment:
I don't see an attribute called `links` on the FileVersion model in the API. This can be removed (and the interface as well).

Reply:
Oh, I read this as if it was defining an attribute field called `links` (e.g. `data.attributes.links`, not the resource links. Ignore this comment!